spec/requests/api/braintree_spec.rb

```diff
@@ -13,11 +13,24 @@
   end
 end
 
+
 describe "Braintree API" do
   def body
     JSON.parse(response.body).with_indifferent_access
   end
 
+  describe 'making a subscription' do
+    let!(:customer) { create(:payment_braintree_customer, email: 'foo@example.com', card_vault_token: '4y5dr6' )}
+
```

Comment:
I'm having to create a `customer`, since recurring depends on there being an existing customer with a payment method   in the vault. 

The `card_vault_token` is real: https://sandbox.braintreegateway.com/merchants/6ngbwhytbhrz9z2r/payment_methods/any/4y5dr6